README.md

```diff
@@ -1,80 +1,98 @@
-Mediated Userspace Device
-=========================
+libvfio-user
+============
 
-Overview
---------
+vfio-user is a framework that allows implementing PCI devices in userspace.
+Clients (such as [qemu](https://qemu.org) talk the [vfio-user
+protocol](https://lists.gnu.org/archive/html/qemu-devel/2020-11/msg02458.html)
+over a UNIX socket to a server. This library, `libvfio-user`, provides an API
+for implementing such servers.
 
-MUSER is a framework that allows implementing PCI devices under the [vfio-user
-protocol](https://lists.gnu.org/archive/html/qemu-devel/2020-11/msg02458.html).
-MUSER is the _backend_ part of the vfio-user protocol, the frontend part is
-implemented by Oracle in https://github.com/oracle/qemu/tree/vfio-user-v0.1.
+[VFIO](https://www.kernel.org/doc/Documentation/vfio.txt) is a kernel facility
+for providing secure access to PCI devices in userspace (including pass-through
+to a VM). With `vfio-user`, instead of talking to the kernel, all operations are
```

Comment:
I think 'all operations' is incorrect. vfio-user do not support iommu groups or any iommu related functionalities which are exposed by vfio over ioctls.

Reply:
That's not really what I meant; I added some clarifying comments.